packages/app/src/app/pages/Sandbox/Editor/Workspace/items/NotOwnedSandboxInfo/NotOwnedSandboxInfo.tsx

```diff
@@ -21,14 +26,17 @@ export const NotOwnedSandboxInfo = () => {
       >
         <Files setEditActions={setEditActions} />
       </WorkspaceItem>
-
       <WorkspaceItem
         defaultOpen
         style={{ marginTop: '.5rem' }}
         title="Dependencies"
       >
         <Dependencies />
       </WorkspaceItem>
+      {console.log(sandbox)}
+      {(sandbox.git || {}).username === user.username && (
```

Comment:
@SaraVieira If you destructur `sandbox` a bit further and give `git` a default value (see above), this line can become

```suggestion
      {username === user.username && (
```